tests/network/bypass/icmp/scenario_09/test.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 
 BUTTERFLY_BUILD_ROOT=$1
-BUTTERFLY_SRC_ROOT=$(cd "$(dirname $0)/../.." && pwd)
+BUTTERFLY_SRC_ROOT=$(cd "$(dirname $0)/../../../../.." && pwd)
```

Comment:
this is one of the first test we made, so I guess the test is simple, yet it's tests after all the complex cases below
This test should be the first icmp test